### ISSUE TITLE
[Bug] 64位版本无法正常编译

### ISSUE THREAD

**Issue by fuchanghao** [Secant1006/PSFExtractor#2]
测试v3.04只能编译32位，64位编译失败

编译环境为
Visual Studio 2022 17.3.2
MSVC v143
Windows SDK for Windows 11 (10.0.22621.1)

编译报错信息：

```
已启动重新生成...
1>------ 已启动全部重新生成: 项目: PSFExtractor, 配置: Release x64 ------
1>pugixml.cpp
1>PSFExtractor.cpp
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(289,45): warning C4244: “参数”: 从“LONGLONG”转换到“ULONG”，可能丢失数据
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(291,40): warning C4244: “参数”: 从“LONGLONG”转换到“DWORD”，可能丢失数据
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(331,15): warning C4244: “初始化”: 从“intmax_t”转换到“int”，可能丢失数据
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(439,60): warning C4244: “参数”: 从“SIZE_T”转换到“DWORD”，可能丢失数据
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(526,88): warning C4267: “参数”: 从“size_t”转换到“ULONG”，可能丢失数据
1>E:\Source_Code\PSFExtractor\src\PSFExtractor.cpp(611,35): warning C4267: “=”: 从“size_t”转换到“int”，可能丢失数据
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 ApplyDeltaB
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 FDIIsCabinet
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 DeltaFree
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 FDICreate
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 FDIDestroy
1>PSFExtractor.obj : error LNK2001: 无法解析的外部符号 FDICopy
1>E:\Source_Code\PSFExtractor\x64\Release\PSFExtractor.exe : fatal error LNK1120: 6 个无法解析的外部命令
1>已完成生成项目“PSFExtractor-cpp.vcxproj”的操作 - 失败。
========== “全部重新生成”: 0 成功，1 失败，0已跳过 ==========
```

**Comment by stdin82** [comment]
Missing for **Release|x64**
`<AdditionalDependencies>Cabinet.Lib;msdelta.lib;%(AdditionalDependencies)</AdditionalDependencies>`

**Comment by Secant1006** [comment]
Fixed in commit d25341b
